client/components/date-range/index.js

```diff
@@ -0,0 +1,665 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { noop, isNil, isNull, has } from 'lodash';
+import { DateUtils } from 'react-day-picker';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { localize, moment } from 'i18n-calypso';
+import DatePicker from 'components/date-picker';
+import Popover from 'components/popover';
+import DateRangeInputs from './inputs';
+import DateRangeHeader from './header';
+import DateRangeTrigger from './trigger';
+import Button from 'components/button';
+import Gridicon from 'gridicons';
+const NO_DATE_SELECTED_VALUE = null;
```

Comment:
A nit here, lets bump this down for a little added clarity:
```suggestion

/**
 * Module variables
 */
const NO_DATE_SELECTED_VALUE = null;
```

Reply:
Fixed